tests/out_test.go

```diff
@@ -303,6 +303,22 @@ var _ = Describe("Out", func() {
 		})
 	})
 
+	Context("when configured with a experimental", func() {
```

Comment:
looks like there's no test for the buildkit param - might need to modify the `docker` stub to dump env so the tests can check against it